example_loop.c fails to include rttest/rttest.h if installed without ament_cmake available

<!--
For general questions, please ask on ROS answers: https://answers.ros.org, make sure to include at least the `ros2` tag and the rosdistro version you are running, e.g. `ardent`.
For general design discussions, please post on discourse: https://discourse.ros.org/c/ng-ros
Not sure if this is the right repository? Open an issue on https://github.com/ros2/ros2/issues
For Bug report or feature requests, please fill out the relevant category below
-->

## Bug report
(Rookie alert - I could just miss the obvious since I am a beginner with Linux)
I followed the installation instruction README.md and build rttest from source with:

```
mkdir build
cd build

cmake ..
sudo make instal
```
I then wanted to build the example by 
```
cd examples
mkdir build
cd build
cmake .. (-DCMAKE_INSTALL_PREFIX=<build folder>)
make
./example_loop
```

However, example_loop.c has an `#include <rttest/rttest.h>`, which did fail in my case unless I changed it to only `#include <rttest.h>`. 
The global install of rttest was targetting `-- Installing: /usr/local/include/rttest.h`.
I did not figure out a working local installation with `-DCMAKE_INSTALL_PREFIX=<build folder>` without changing example_loop.c .




**Required Info:**

- Operating System:
 Ubuntu 16.04 (4.14.93-rt53 RT_Preempt patched)
- Installation type:
  - from source
- Version or commit hash:
  - cd3a2d801909523996b3a236f800d9e94df5d5fb
- DDS implementation:
  -  RTI Connext
- Client library (if applicable):
  - N/A

#### Steps to reproduce issue
<!-- Detailed instructions on how to reliably reproduce this issue http://sscce.org/
``` code that can be copy-pasted is preferred ``` -->
```
git clone https://github.com/ros2/realtime_support.git

cd realtime_support
cd rttest

mkdir build
cd build

cmake ..
sudo make install

cd ..
cd examples
mkdir build
cd build
cmake .. 

```

#### Expected behavior
```
Scanning dependencies of target example_loop
[ 50%] Building C object CMakeFiles/example_loop.dir/example_loop.c.o
[100%] Linking C executable example_loop
[100%] Built target example_loop
```

#### Actual behavior

```
Scanning dependencies of target example_loop
[ 50%] Building C object CMakeFiles/example_loop.dir/example_loop.c.o
/home/nicolasgerig/realtime_support/rttest/examples/example_loop.c:17:27: fatal error: rttest/rttest.h: No such file or directory
compilation terminated.
CMakeFiles/example_loop.dir/build.make:62: recipe for target 'CMakeFiles/example_loop.dir/example_loop.c.o' failed
make[2]: *** [CMakeFiles/example_loop.dir/example_loop.c.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/example_loop.dir/all' failed
make[1]: *** [CMakeFiles/example_loop.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```


#### Additional information

<!-- If you are reporting a bug delete everything below
     If you are requesting a feature deleted everything above this line -->
----
## Feature request
Update README or correct the example_loop.c?

#### Feature description
<!-- Description in a few sentences what the feature consists of and what problem it will solve -->

#### Implementation considerations
<!-- Relevant information on how the feature could be implemented and pros and cons of the different solutions -->

--- a/rttest/CMakeLists.txt
+++ b/rttest/CMakeLists.txt
@@ -81,7 +81,9 @@ else()
     ARCHIVE DESTINATION ${INSTALL_LIB_DIR}
   )
 
-  install(FILES ${PROJECT_SOURCE_DIR}/include/rttest/rttest.h ${PROJECT_SOURCE_DIR}/include/rttest/utils.h DESTINATION ${INSTALL_INCLUDE_DIR})
+  install(
+    DIRECTORY include/
+    DESTINATION ${INSTALL_INCLUDE_DIR})
 
   export(PACKAGE ${PROJECT_NAME})
 

